metaseq/tasks/streaming_language_modeling.py

```diff
@@ -411,7 +411,7 @@ def load_dataset(self, split: str, epoch=1, combine=False, **kwargs):
             else:
                 metric_df = FilterDataset.retrieve_metric_df(
                     metric_file=self.args.use_data_pruning_metrics_filepath,
-                    dataset_name_to_index=dataset_name_to_index,
```

Comment:
thanks for changing the name, i have no idea why i called it dataset_name_to_index here lol